contents/docs/how-posthog-works/queries.mdx

```diff
@@ -21,12 +21,13 @@ As an example, let's say that we have the following list of events:
 
 In this case, if we ran a query asking for the number of unique users who viewed a page, we would get a result of `2`, as our table contains 2 unique `person_id`'s.
 
-The way that we write the `person_id` to each event has some implications for the number of unique users that are displayed:
+The way we write the `person_id` to each event has some implications for the number of unique users that are displayed:
 
 1. Some users are counted twice on the trend graph. 
-   To make queries fast we only look at the events table. However, it is not possible to determine whether two `person_id`'s were later merged into a single user, which results in them being counted separately.
+   The source of truth for data is the events table. Since this is point-in-time data, it is not possible to determine whether two `person_id`'s were later merged into a single user, which results in them being counted separately.
+   > Note: This was never an issue pre PostHog version 1.39 because we'd join the latest person info to get this result. However, this doesn't scale well, and we've decided to not continue going down this route.
 2. In the person modal, the count may be lower than the count displayed in the graph. 
-   To make retrieving the information for the person modal fast, we ignore persons who have been merged (and hence deleted).
+   Persons who've been merged into one have one of their old IDs deleted. We remove these people from the persons modal, as there's no place to link them to.
```

Comment:
Do we remove them both? persons and people across these two sentences throws me :)

Something more concrete like this maybe???

If person A is merged into person B we don't show person A in the persons modal. Because it only considers the point-in-time data it's not possible to determine that person A should link to person B

Reply:
you can see the example below for more elaboration - only `user-2` is deleted, we keep `user-1`

Reply:
Ah, might be an artefact of reading it in the PR view instead of as an article :)